test/e2e/network/service.go

```diff
@@ -2338,14 +2338,13 @@ func execSourceipTest(pausePod v1.Pod, serviceAddress string) (string, string) {
 
 	framework.ExpectNoError(err)
 
-	// The stdout return from RunHostCmd seems to come with "\n", so TrimSpace is needed.
-	// Desired stdout in this format: client_address=x.x.x.x
-	outputs := strings.Split(strings.TrimSpace(stdout), "=")
-	if len(outputs) != 2 {
+	// The stdout return from RunHostCmd is in this format: x.x.x.x:port or [xx:xx:xx::x]:port
+	host, _, err := net.SplitHostPort(stdout)
+	if err != nil {
 		// ginkgo.Fail the test if output format is unexpected.
-		e2elog.Failf("exec pod returned unexpected stdout format: [%v]\n", stdout)
+		e2elog.Failf("exec pod returned unexpected stdout: [%v]\n", stdout)
 	}
-	return pausePod.Status.PodIP, outputs[1]
+	return pausePod.Status.PodIP, output
```

Comment:
er you don't need to use the suggestion to commit that change, but you do need to update this line here to use the new name :sweat_smile:

Reply:
that´s for trying to fix things from my mobile :)